Magento2/Sniffs/CodeAnalysis/EmptyBlockSniff.php

```diff
@@ -25,4 +26,19 @@ public function register()
             ]
         );
     }
+    /**
+     * @inheritDoc
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $posOfFunction = $phpcsFile->findNext([T_FUNCTION], $stackPtr);
```

Comment:
This will cause false-positive findings, because there might be a lot of `T_FUNCTION ` tokens in the file. We need to check the content of `$stackPtr` token.